beep/features/featurizer_helpers.py

```diff
@@ -415,13 +420,13 @@ def get_chosen_df(processed_cycler_run, diag_pos):
     return chosen
 
 
-def res_calc(chosen, diag_pos, soc, step_ocv, step_cur, index):
+def res_calc(chosen, steps, soc, step_ocv, step_cur, index):
```

Comment:
While you are on this, can we rename `chosen` to something more descriptive, say `df_diag` or such?

Reply:
done